src/leetCodeExecutor.ts

```diff
@@ -100,6 +110,19 @@ class LeetCodeExecutor {
         }
     }
 
+    public async getCompaniesAndTags(): Promise<{ companies: { [key: string]: string[] }, tags: { [key: string]: string[] } }> {
+        // preprocess the plugin source
+        const componiesTagsPath: string = path.join(await leetCodeExecutor.getLeetCodeRootPath(), "lib", "plugins", "company.js");
+        const componiesTagsSrc: string = (await fs.readFileSync(componiesTagsPath, "utf8")).replace(
```

Comment:
We can use `readFile` from `fs-extra`.

Reply:
Something went wrong with my usage. `fs-extra` is correct.